src/otx/recipe/_base_/data/anomaly.yaml

```diff
@@ -0,0 +1,76 @@
+task: ANOMALY_CLASSIFICATION
+input_size: 256
+data_format: mvtec
+mem_cache_size: 1GB
+mem_cache_img_max_size: null
+image_color_channel: RGB
+stack_images: false
+unannotated_items_ratio: 0.0
+train_subset:
+  subset_name: train
+  transform_lib_type: TORCHVISION
+  to_tv_image: true
+  batch_size: 32
+  num_workers: 4
+  transforms:
+    - class_path: otx.core.data.transform_libs.torchvision.ResizetoLongestEdge
+      init_args:
+        size: $(input_size)
+        antialias: true
+    - class_path: otx.core.data.transform_libs.torchvision.PadtoSquare
```

Comment:
Could you check if these transforms can be changed to `Resize` and `Pad` respectively?

Reply:
@ashwinvaidya17 Can you please check on this?